tests/nlp/conftest.py

```diff
@@ -38,6 +38,14 @@ def tweet_emotion_train_test_textdata():
                                           include_embeddings=True)
     return train, test
 
+@pytest.fixture(scope='session')
+def just_dance_train_test_textdata_sampled():
+    """Just Dance text multilabel classification dataset"""
+    train, test = just_dance_comment_analysis.load_data(data_format='TextData', as_train_test=True)
```

Comment:
Consider saving separate fixture for testing... suspect that takes a lot of time in CI

Reply:
this is a fixture aimed to save run time

Reply:
Sorry I ment create a version of the dataset in which we don't download all the just dance data, but only a small sample

Reply:
Not critical, just thinking about the ever growing CI time...